scripts/onboarding/aks/onboarding-msi-bicep/nested_aks_monitoring_msi_dcra_aksResourceId.bicep

```diff
@@ -0,0 +1,13 @@
+@description('ContainerInsights extension association name')
+param associationName string 
+
+@description('Data collection rule Id')
+param dataCollectionRuleId string 
+
+resource clusterName_microsoft_insights_association 'microsoft.insights/dataCollectionRuleAssociations@2021-04-01' = {
```

Comment:
update with latest API version  2022-06-01 instead of 2021-04-01

Reply:
changed